contrib/esp-sync.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+ESPS=(
+  "/dev/sdb1"
+  "/dev/sdc1"
+  "/dev/sdd1"
+)
+
+BMP="$( yq-go eval ".Global.BootMountPoint" /etc/zfsbootmenu/config.yaml )"
+if [ -z "${BMP}" ]; then
+  echo "Unable to determine BootMountPoint"
+  exit 1
+fi
+
+IMG_DIR="$( yq-go eval ".EFI.ImageDir" /etc/zfsbootmenu/config.yaml )"
+if [ -z "${IMG_DIR}" ]; then
+  echo "Unable to determine ImageDir"
+  exit 1
+fi
+
+IMG_REL="${IMG_DIR#"${BMP}"}"
+
+mount "${BMP}"
+
+if ! ESP_MNT="$( mktemp -d )"; then
+  echo "Unable to create temporary mountpoint"
+  exit
+fi
+
+trap 'rmdir ${ESP_MNT}' EXIT INT TERM
```

Comment:
You could add a check that ESP_MNT is a mount point and I mount if so. That should clean up if the script is interrupted between mount/unmount in the loop.

Reply:
good call, done!